### PR TITLE
Added alias method to help with QRCode printing

### PR DESCRIPTION
**Pull request by igorocampos**
This can make it easy for first-comers that want to use the library for QRCode printing. Realized that during an issue discussion.

I also realized that the project doesn't have any unit tests. Maybe it would be another good thing to help with automated regression tests. @lukevp what do you think about it?

For example, I could do an Assert on this new method for whenever a type different from QRCodes is informed, the method throws the ArgumentException. This way, in the future, if for any reason this stops to happen we can easily see when and fix it.

### CODE REVIEW

File: ESCPOS_NET.ConsoleTest/Test2DCodes.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public static partial class Tests
             e.PrintLine(),
 
             e.PrintLine("QRCODE MODEL 2:"),
-            e.Print2DCode(TwoDimensionCodeType.QRCODE_MODEL2, websiteString),
+            e.Print(websiteString),
```
Comment (lukevp) [inline ESCPOS_NET.ConsoleTest/Test2DCodes.cs:28]: I think this should be PrintQRCode

**Review by lukevp — COMMENTED** [review]
This is a nice improvement, thanks Igor.  Just 1 comment.

I would love some unit tests on the project, I didn't need them at the time because I was interactively testing on the printer, so you can consider the ConsoleTest as an integration test suite.  The main issue with the unit test design will be what is under test, and how we can properly assert failures.  Should we test for specific byte strings on the emitters? Which sections should have test cases built first?  Why don't you open an issue where we can discuss the unit tests and add them to the project/wiki for coordination?

File: ESCPOS_NET.ConsoleTest/Test2DCodes.cs (hunk 1 above)
Comment (igorocampos) [reply ESCPOS_NET.ConsoleTest/Test2DCodes.cs:28]: yep yep yep. Sorry about that, missed it.

Comment (igorocampos) [reply ESCPOS_NET.ConsoleTest/Test2DCodes.cs]: Fixed in 8abffce68f711bf4128c207407e49e5e1fe01b1b

**Pull request merged**